torch/_torch_docs.py

```diff
@@ -1482,6 +1482,134 @@ def merge_dicts(*dicts):
     silently incorrect.
 """)
 
+add_docstr(torch.hsplit,
+           r"""
+hsplit(input, indices_or_sections) -> List of Tensors
+
+Splits :attr:`input`, a tensor with one or more dimensions, into multiple tensors
+horizontally (column wise) according to :attr:`indices_or_sections`. Each split
+is a view of :attr:`input`.
+
+If :attr:`input` is one dimensional this is equivalent to calling
+torch.tensor_split(input, indices_or_sections, dim=0) (the split dimension is
+zero), and if :attr:`input` has two or more dimensions it's equivalent to calling
+torch.tensor_split(input, indices_or_sections, dim=1) (the split dimension is 1),
+except that if :attr:`indices_or_sections` is an integer it must evenly divide
+the split dimension or a runtime error will be thrown.
+
+This function is based on NumPy's :func:`numpy.hsplit`.
+
+Args:
+    input (Tensor): tensor to split.
+    indices_or_sections (int or list(ints) or tensor of integers): See argument in :func:`torch.tensor_split`.
```

Comment:
This should be consistent with the documentation for tensor_split:

https://github.com/pytorch/pytorch/blob/8c74e1b840003832cb0a3fb84ee065a508620d70/torch/_torch_docs.py#L1388

Reply:
(this comment should also be applied to the docs for the other functions)